python/instrumentation/openinference-instrumentation-vertexai/examples/audio_transcription.py

```diff
@@ -0,0 +1,26 @@
+import vertexai
+from openinference.instrumentation.vertexai import VertexAIInstrumentor
+from opentelemetry.exporter.otlp.proto.grpc.trace_exporter import OTLPSpanExporter
+from opentelemetry.sdk.trace import TracerProvider
+from opentelemetry.sdk.trace.export import ConsoleSpanExporter, SimpleSpanProcessor
+from vertexai.generative_models import GenerativeModel, Part
+
+endpoint = "http://127.0.0.1:4317"
+tracer_provider = TracerProvider()
+tracer_provider.add_span_processor(SimpleSpanProcessor(OTLPSpanExporter(endpoint)))
+tracer_provider.add_span_processor(SimpleSpanProcessor(ConsoleSpanExporter()))
+
+VertexAIInstrumentor().instrument(tracer_provider=tracer_provider)
+
+vertexai.init(location="us-central1")
+model = GenerativeModel("gemini-1.5-flash")
+prompt = """Can you transcribe this interview, in the format of timecode, speaker, caption.
+Use speaker A, speaker B, etc. to identify speakers.
+"""
+audio_file_uri = "gs://cloud-samples-data/generative-ai/audio/pixel.mp3"
```

Comment:
is this intended for future use-cases?

Reply:
Technically, we can also just convert these in phoenix or the instrumentor, i.e. 

https://storage.googleapis.com/cloud-samples-data/generative-ai/audio/pixel.mp3

But I can also take these examples out if that's better.